sdks/v4-sdk/src/PositionManager.ts

```diff
@@ -96,11 +105,6 @@ export interface RemoveLiquiditySpecificOptions {
    * The optional permit of the token ID being exited, in case the exit transaction is being sent by an account that does not own the NFT
    */
   permit?: NFTPermitOptions
-
-  /**
-   * Parameters to be passed on to collect
-   */
-  collectOptions: Omit<CollectSpecificOptions, 'tokenId'>
```

Comment:
Hmm curious why we remove this? do we not collect fees when we remove in v4?

Reply:
Fees are auto collected when you decrease & take together